src/RPCServer.ts

```diff
@@ -3,11 +3,11 @@ import type {
   IdGen,
   ClientHandlerImplementation,
   DuplexHandlerImplementation,
-  JSONRPCError,
+  JSONRPCResponseError,
   JSONRPCRequest,
   JSONRPCResponse,
-  JSONRPCResponseError,
-  JSONRPCResponseResult,
+  JSONRPCResponseFailed,
+  JSONRPCResponseSuccess,
```

Comment:
Why are these renamed? From what I recall they were named based on the RPC spec before.
https://www.jsonrpc.org/specification#response_object

But I suppose it doesn't matter too much.

Reply:
@amydevs?